rebasehelper/specfile.py

```diff
@@ -102,6 +103,85 @@ def get_strip(self):
         return self.strip
 
 
+class SpecContent(object):
+
+    """Class representing content of a SPEC file."""
+
+    defined_section_headers = [
```

Comment:
I'd use `known_section_headers` or maybe just `section_headers`. Also, if I'm not mistaken, in most other classes *class constants* are uppercase.